src/app.py

```diff
@@ -7,7 +8,10 @@
 
 def main():
     app = QApplication(sys.argv)
-    window = MainWindow()  # noqa
+    data = open('configuration.json')
+    config = json.load(data)
+    window = MainWindow(config)  # noqa
```

Comment:
```suggestion
    with open('configuration.json', 'r') as f:
        config = json.load(f)
    window = MainWindow(config)  # noqa
```
`with` statement ensures that file will be closing after reading